docker-compose.yaml

```diff
@@ -38,12 +38,12 @@ services:
             DEFAULT_LOCALE:
             SERVICE_NAME:
             DATABASE_URL: pgsql://postgres:@db:5432/postgres
-            ASSETS_URI: http://localhost:8081/bucket/
```

Comment:
This change means the URI in the data will be `http://s3:9000` rather than something accessible from the host. Not a problem for real usage, though means you couldn't use CloudFront.